src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -86,6 +86,7 @@ const TooltipRenderedOnPageBody = (props) => {
     const {animationStyle, rootWrapperStyle, textStyle, pointerWrapperStyle, pointerStyle} = useMemo(
         () =>
             getTooltipStyles(
+                rootWrapper.current,
```

Comment:
@blazejkustra I saw that you used `RefObject<View>` for the `tooltip` parameter.

```ts
    interface RefObject<T> {
        readonly current: T | null;
    }
```

It is expecting the ref who has `current`, but you are passing only the `current`.

Reply:
Actually tooltip isn't used at all, so I'll remove the argument.